### PR TITLE
[OBSDEF-9632] Rename federation to objectscale-federation

### PR DESCRIPTION
**Pull request by voa808**
## Purpose
[OBSDEF-9632] Rename federation to objectscale-federation

## PR checklist
- [ ] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
make build - Passed in DevKit

Testing is done using custom images of object-services, operator, charts, iamsvc, management gateway images AND service console, automation and charts branch here:
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/fedsvc-custom-ci/171/
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/fedsvc-custom-acceptance/45/

In progrees
https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/258/console

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: objectscale-manager/Chart.yaml [hunk 1]
```diff
@@ -27,9 +27,13 @@ dependencies:
     version: 0.77.0
     repository: file://../objectscale-gateway
     condition: objectscale-gateway.enabled
-  - name: federation
+  - name: objectscale-federation
+<<<<<<< HEAD
```
Comment (SeemaTahaliyani) [inline objectscale-manager/Chart.yaml]: @voa808 please resolve this conflict, other than everything looks good.

Comment (voa808) [reply objectscale-manager/Chart.yaml]: Fixed.

**Review by SeemaTahaliyani — APPROVED** [review]
LGTM

**Comment by voa808** [comment]
Unable to run custom-ci job. I checked charts-master-ci job as well which shows similar failure message. 
`Setup failed: ECS Flex should be installed`

**Comment by voa808** [comment]
charts-ci works on Stonecold, which is blocked by https://jira.cec.lab.emc.com/browse/LABDEF-584

**Review by hldnova — COMMENTED** [review]
should objectscale-federation fields be updated to be consistent with iam/dcm?, e.g.,
`objectscale-federation.fedsvc.image.tag` to `objectscale-federation.image.tag`
`objectscale-federation.fedsvc.replicaCount` to `objectscale-federation.replicaCount`

**Comment by voa808** [comment]
> should objectscale-federation fields be updated to be consistent with iam/dcm?, e.g.,
> `objectscale-federation.fedsvc.image.tag` to `objectscale-federation.image.tag`
> `objectscale-federation.fedsvc.replicaCount` to `objectscale-federation.replicaCount`

Sure @hldnova We can definitely make the change in another PR.

**Review by eddiepavkovic — APPROVED** [review]
lgmt

**Comment by voa808** [comment]
> > should objectscale-federation fields be updated to be consistent with iam/dcm?, e.g.,
> > `objectscale-federation.fedsvc.image.tag` to `objectscale-federation.image.tag`
> > `objectscale-federation.fedsvc.replicaCount` to `objectscale-federation.replicaCount`
> 
> Sure @hldnova We can definitely make the change in another PR.

OBSDEF-10345 is created for the same.

**Pull request merged**